src/components/Footer.tsx

```diff
@@ -41,14 +41,13 @@ const Footer = () => {
                         <Grid item xs={12} sm={3} lineHeight={2}>
                             <Box sx={styles.bottomLine}>CONTACT</Box>
                             <Box>
-                                <Link
-                                    href="/contact"
+                                <RouteLink
+                                    to="/contact"
                                     target="_blank"
-                                    underline="none"
-                                    sx={styles.muiLink}
+                                    style={styles.routeLink}
                                 >
                                     Contact Us
-                                </Link>
+                                </RouteLink>
                             </Box>
                             <Box>Email: devprep@outlook.com.au</Box>
```

Comment:
I would suggest removing these Boxes as the Contact Us page already contains the informations:
`<Box>Email: devprep@outlook.com.au</Box>`
```
<Box>
   <Link
    href="https://github.com/DevPreps"
    target="_blank"
    rel="noreferrer"
    underline="none"
    sx={styles.muiLink}
    >
       Github
   </Link>
</Box>
```
Or you can leave the GitHub link.
Also, please fix the spelling for Privacy from Privary :)

Reply:
Thanks for the reminder, you are right. The email and GitHub those boxes have been removed, and the wrong spelling has been corrected.